helm/templates/_helper_logger_probes.tpl

```diff
@@ -0,0 +1,18 @@
+{{- define "logger.probes" }}
+livenessProbe:
+  exec:
+    command: ["ash", "-c", 'expr $(pgrep ruby | wc -l) ">=" 2']
```

Comment:
What is `ash`? Any reference link?

Reply:
it's the shell which is available in the logger docker image (instead of bash)